File: __Aulas_2024/ListaCursoVip4/app/src/main/java/devandroid/moacir/listacursovip4/view/MainActivity.java
package devandroid.moacir.listacursovip4.view;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import devandroid.moacir.listacursovip4.R;
import devandroid.moacir.listacursovip4.controller.PessoaController;
import devandroid.moacir.listacursovip4.model.Pessoa;

public class MainActivity extends AppCompatActivity {
    //declaração das classes

    Pessoa pessoa;
    PessoaController controller;

    EditText editPrimeiroNome;
    EditText editSobreNomeAluno;
    EditText editNomeCurso;
    EditText editTelefoneContato;
    Button btnLimpar;
    Button btnSalvar;
    Button btnFinalizar;

    String dadosPessoa;

       @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });

       controller = new PessoaController();
       controller.toString();

       pessoa = new Pessoa();
       pessoa.setPrimeiroNome("Joao");


       }
}

File: __Aulas_2024/POOHerancaKotlin/app/src/main/java/devandroid/moacir/pooherancakotlin/MainActivity.kt
package devandroid.moacir.pooherancakotlin

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        lateinit var Joao: Funcionario
        Joao.setNome("Joao")
        Joao.setSalario(1200)
        Joao.setDataAdmissao("01/02/2024")

        lateinit var gerentePedro: Gerente

        gerentePedro.setNome("Pedro")
        gerentePedro.setDataAdmissao("01/01/2022")
        gerentePedro.setSalario(2400)
        gerentePedro.setComissao(10.5)
        gerentePedro.setTotalFuncionario(25)







    }
}